webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path')
+const HtmlWebPackPlugin = require('html-webpack-plugin')
+
+module.exports = {
+  entry: './src/index.tsx',
+  output: {
+    path: path.join(__dirname, '/dist'),
+    filename: 'index_bundle.js',
+    chunkFilename: '[name].[chunkhash].js',
+    assetModuleFilename: 'images/[hash][ext][query]'
+  },
+  devtool: 'source-map',
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader'
+        }
+      },
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/
+      },
+      {
+        test: /\.css$/i,
+        use: [
+          'style-loader',
+          {
+            loader: 'css-loader',
```

Comment:
It's better to have optimization for CSS and JS, please take you time for finding information how to minimize and optimize your bundles